src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -52,6 +53,15 @@
 LOG_RUNNING_PROMPT = "This command usually takes minutes to run. Add '--verbose' parameter if needed."
 
 
+class ApplicationInsightsCredential():
+    def __init__(self):
+        self.connection_string = None
+        self.instrumentation_key = None
+
+    def is_valid(self):
+        return (self.connection_string is not None) and (self.instrumentation_key is not None)
+
```

Comment:
nit: duplicated empty line.

Reply:
This `custom.py` file is really too long, current code format is use 2 empty lines to separate the functions/methods.

Reply:
Can we file one bug to track the `too long` and sync with shichao for this ? It looks not the right way here.

Reply:
Sure